specs/experimental/standard-l2-genesis.md

```diff
@@ -180,10 +198,18 @@ event TransactionDeposited(address indexed from, address indexed to, uint256 ind
 The following fields are included:
 
 - `from` is the `DEPOSITOR_ACCOUNT`
-- `to` is `Predeploys.ProxyAdmin`
+- `to` is `Predeploys.L2_PROXY_ADMIN`
 - `version` is `uint256(0)`
 - `opaqueData` is the tightly packed transaction data where `mint` is `0`, `value` is `0`, the `gasLimit`
-   is `200_000`, `isCreation` is `false` and the `data` is the data passed into `upgrade`.
+  is `200_000`, `isCreation` is `false` and the `data` is the data passed into `upgrade`.
```

Comment:
the gasLimit presumably needs to be updated if it becomes an input.

I'm also curious what the current thinking is around the correct behaviour of `useGas()`:

![CleanShot 2025-02-19 at 15 56 14@2x](https://github.com/user-attachments/assets/619d6f39-764f-435a-8f63-eb714fe3621e)

Note that I don't agree with the recommended fix, as it will enables anyone to DoS a system deposit.

Reply:
We believe that we shouldnt add the check in `useGas` since this can cause further DoS issues by front-running the chain-operator. To compensate for this we agree with adding a test to make sure we dont exceed the buffer gas limit like mentioned here https://github.com/ethereum-optimism/design-docs/pull/97#discussion_r1811253276